### DISCUSSION TITLE
Welcome to TeleDrive Discussions!

### DISCUSSION THREAD

**Discussion by mgilangjanuar** [mgilangjanuar/teledrive#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fadelrm** [comment]
bang gabisa login

**Comment by denomanoamedi** [comment]
Hi, I'm a beginner at this so please don't mind. 
I was following the deployment videos on youtube but it seems that Heroku is no longer free or it at least asks for a credit card (i don't have any access to a credit card), so my request is if possible can you guys share a tutorial where there is no payment required as such.
